src/bundle/Resources/views/content/tabs/relations/table.html.twig

```diff
@@ -0,0 +1,40 @@
+{% trans_default_domain "locationview" %}
+
+{% if relationList %}
+    <table class="ez-table-data">
+        <thead>
+            <tr>
+                <th>{{ 'locationview.relations.name'|trans()|desc('Name') }}</th>
+                <th>{{ 'locationview.relations.content.type'|trans()|desc('Content type') }}</th>
+                <th>{{ 'locationview.relations.relation.type'|trans()|desc('Relation type') }}</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for relation in relationList %}
+            <tr>
+                <td><a href="{{ path(relation.destinationLocation) }}">{{ relation.destinationContentInfo.name }}</a></td>
+                <td>{{ relation.destinationContentTypeName }}</td>
+                <td>
+                    {% if relation.type == constant('eZ\\Publish\\API\\Repository\\Values\\Content\\Relation::COMMON') %}
```

Comment:
What about this:

```
{% for relation in relationList %}
<tr>
    <td><a href="{{ path(relation.destinationLocation) }}">{{ relation.destinationContentInfo.name }}</a></td>
    <td>{{ relation.destinationContentTypeName }}</td>
    <td>{{ ez_ui_relation_type(relation) }}</td>
</tr>
{% endfor %}        
```

The Twig function does the type checking, and uses the translation service to return the translated version.